api/api/src/test/scala/uk/ac/wellcome/platform/api/services/WorksServiceTest.scala

```diff
@@ -338,6 +338,66 @@ class WorksServiceTest
         result.left.get shouldBe a[ElasticError]
       }
     }
+
+    // See: https://www.elastic.co/guide/en/elasticsearch/reference/current/query-dsl-simple-query-string-query.html#simple-query-string-syntax
+    describe("simple query string syntax") {
+      it("uses none but PHRASE simple query syntax") {
+        val work = createIdentifiedWorkWith(
+          title =
+            "+a -title | with (all the simple) query~4 syntax operators in it*"
+        )
+
+        // prefix
+        assertSearchResultIsCorrect(query =
+          "+a -title | with (all the simple) query~4 syntax operators in it*")(
+          allWorks = List(work),
+          expectedWorks = List(work),
+          expectedTotalResults = 1
+        )
+      }
+
+      it("doesn't throw a too_many_clauses exception when passing an invalid simple query syntax query") {
+        val workEmu = createIdentifiedWorkWith(
+          title = "a b c"
+        )
+
+        assertSearchResultIsCorrect(
+          query = "(a b c d e) h"
```

Comment:
Why is this query invalid?

Reply:
Says in the description, this would normally throw a `too_many_clauses` error.

Reply:
updated description